build-tools/packages/build-cli/src/library/repoPolicyCheck/dockerfilePackages.ts

```diff
@@ -26,23 +27,30 @@ function getOrAddLocalMap(key: string, getter: () => Buffer): Buffer {
 export const handler: Handler = {
 	name: "dockerfile-packages",
 	match: /^(server\/routerlicious\/packages)\/.*\/package\.json/i,
-	handler: async (file: string): Promise<string | undefined> => {
+	handler: async (file: string, gitRoot: string): Promise<string | undefined> => {
+		// const relativePath = path.relative(process.cwd(), file);
 		// strip server path since all paths are relative to server directory
-		const dockerfileCopyText = getDockerfileCopyText(file.replace(serverPath, ""));
-
+		const dockerfileCopyText = getDockerfileCopyText(
+			path.relative(gitRoot, file).replace(serverPath, ""),
+		);
```

Comment:
In the resolver when we call `getDockerfileCopyText()` we only pass the repo-relative path without removing the `serverPath`; this should be the same, no?

Reply:
yes, good catch. I wish resolvers and handlers could be a single thing so we could more easily share logic.

Reply:
To me it's always felt like a handler + resolver combo should be an implementation of an abstract class or an interface that we can implement with a class to encapsulate and share logic between the two. Maybe an FHL project for some day.